pkg/devfile/parser/data/v2/commands_test.go

```diff
@@ -191,13 +278,19 @@ func TestDevfile200_UpdateCommands(t *testing.T) {
 
 			d.UpdateCommand(tt.newCommand)
 
-			commandsMap := d.GetCommands()
+			commands, _ := d.GetCommands(common.DevfileOptions{})
```

Comment:
same as above, the error should not be ignored. should check unexpected error

Reply:
same as above

Reply:
updated